content/docs/manuals/networking-internals/load-balancing.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Load Balancing"
+summary: ""
+draft: false
+weight: 314
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+The load balancing in Oakestra consists of a multistep address conversion protocol using semantic addresses as part
+of an overlay network.
+
+### Semantic Addressing
+
+The semantic addressing happens on the service layer.
+Similar to a cluster IP in Kubernetes, these addresses reference all the instances (replicas) of a microservice
+with a single address. This address does not change when scaling up the instances or when migrating them.
+Unlike the Kubernetes cluster IP, when deploying a service in Oakestra, the platform provides as many Service IP
+addresses as the number of balancing policies supported (and active) within the platform.
+
+#### Example
+
+{{<svg "network-example-dual-stack" >}}
+
+A developer can communicate with any instance of Service B either with **Round Robin** balancing policy or
+the **Closest** balancing policy. The former balances the traffic evenly between all the instances, and the
+latter finds the geographically closer instance.
+
+Click on the following tabs to see how every request is handled.
+
+{{< tabs-icon "Requests" >}}
+{{< tab-icon "First" "_envelope_1.png">}}
+Service A performs the first request using the ServiceIP `fdff:1000::1` representing the closest instance balancing policy.
+The network components' proxy converts the address to the Namespace IP of Service B Instance 1, which looks like it is
+the geographically closer service. The message will be, therefore **transparently** delivered to the closest instance
+of Service B. Note that the Namespace IP is the real address of the instance, the one provisioned at deployment time,
+unique and dynamic. An application developer never sees or uses this address, as it depends on the subnetwork of the
+specific machine where the service is deployed.
+{{< /tab-icon >}}
+
+{{< tab-icon "Second" "_envelope_2.png">}}
+Service A performs a second request using the ServiceIP `fdff:2000::1` representing the Round Robin balancing
+policy. The network components' proxy converts the address to the Namespace IP of Service B Instance 3, which is
+randomly chosen among all the available instances.
+{{< /tab-icon >}}
+
+{{< tab-icon "Third" "_envelope_3.png">}}
```

Comment:
envelope color does not match that of dual-stack fig

Reply:
Are you sure? You may need to clear your cache, as the file has changed, but not the name. On my end it seems fine.